.env

```diff
@@ -38,4 +38,5 @@ STRIPE_RESPONSE_URL=null
 STRIPE_DEFERRED_INTENT_BETA_FLAG=elements_enable_deferred_intent_beta_1
 SUBSCRIPTIONS_BASE_URL=null
 ENABLE_B2C_SUBSCRIPTIONS=false
-COMMERCE_COORDINATOR_BASE_URL=null
\ No newline at end of file
+COMMERCE_COORDINATOR_BASE_URL=null
+WAFFLE_FLAGS=""
```

Comment:
We should note somewhere that waffle flags coming via query parameters override any `WAFFLE_FLAGS` set here.

Reply:
agreed, no docs have been updated. with this work. There is a CC side ticket for this, ill mod it to be both.

Reply:
[THES-242](https://2u-internal.atlassian.net/browse/THES-242)